imports/components/giving/add-to-cart/index.js

```diff
@@ -296,14 +307,15 @@ class CartContainer extends Component {
     if (state.subfunds.length === 2) {
       this.props.clearTransaction(state.subfunds[1].fundId);
     }
-  }
+  };
 
-  setCanCheckout = (canCheckout: boolean) => this.setState({ canCheckout })
+  setCanCheckout = (canCheckout: boolean) => this.setState({ canCheckout });
 
   canCheckout = (total: number) => {
     if (total <= 0) return false;
-    return this.state.canCheckout;
-  }
+    return true;
+    // return this.state.canCheckout;
+  };
```

Comment:
@delianides sounds great!

@richarddubay this is good to update snapshots and deploy to beta